src/components/Overview.tsx

```diff
@@ -178,6 +179,17 @@ export default class Overview extends Component<OverviewProps, OverviewState> {
 		refreshKey: 1,
 	}
 
+	componentWillMount(): void {
```

Comment:
`compnentWillMount` je už deprecated.

Nicméně můžeš místo téhle metody to rovnou dát do té `state` property výše...

Myslím, že takhle by to šlo:

```tsx
export default class Overview extends Component<OverviewProps, OverviewState> {
	state: OverviewState = {
		searchTags: [],
		platform: initialPlatform,
		sort: {
			value: SortValue[browserStorage.get('overview.sort.value', SortValue.DATE).toUpperCase()],
			direction: SortDirection[browserStorage.get('overview.sort.direction', SortDirection.DESC).toUpperCase()],
		},
		refreshKey: 1,
	}
	// ...
}
```

Reply:
@ViliamKopecky done